.travis.yml

```diff
@@ -8,12 +8,21 @@ notifications:
     on_failure: always
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
   - 7.0
-  - hhvm
+  - 7.1
+  - 7.2
+
+matrix:
```

Comment:
I'm happy to drop support for PHP5.3 and HHVM if they are causing problems.

Reply:
all problems are resolved already. :)
working on adding 'files' key. Will be resolved soon